docs/source/getting-started.rst

```diff
@@ -1,48 +1,246 @@
 Getting Started
 ===============
 
-Perform the following steps on your laptop or development machine
+It only takes a few minutes to get StreamAlert up and running!
 
-Dependencies
-------------
+Install Dependencies
+--------------------
 
 1. Install Python 2.7 and `pip <https://pip.pypa.io/en/stable/installing/>`_
-2. Install `Terraform <https://www.terraform.io/intro/getting-started/install.html>`_
-3. Download StreamAlert:
+2. Install `Terraform <https://www.terraform.io/intro/getting-started/install.html>`_ v0.11.X:
 
 .. code-block:: bash
 
-  $ git clone --branch stable https://github.com/airbnb/streamalert.git
+  brew install terraform  # MacOS Homebrew
+  terraform --version     # Must be v0.11.X
 
-4. Install and activate Virtualenv:
+3. Install `virtualenv <https://virtualenv.pypa.io/en/stable/installation/>`_:
 
 .. code-block:: bash
 
-  $ cd streamalert
-  $ pip install --user virtualenv
-  $ virtualenv -p python2.7 venv
-  $ source venv/bin/activate
+  pip install --user virtualenv
+  virtualenv --version
 
-5. Install dependencies:
+4. If you're on a Linux system, you may need to install the Python development libraries:
 
 .. code-block:: bash
 
-  $ pip install -r requirements.txt
+  sudo apt install python-dev    # Debian
+  sudo yum install python-devel  # CentOS/RHEL
 
+Download StreamAlert
+--------------------
 
-.. note:: If you run into issues with ``psutil`` installation, make sure you have the ``python-dev`` dependencies installed.  `debian: sudo apt install python-dev``, `CentOS/RHEL: sudo yum install python-devel``
+1. Clone the latest stable release of StreamAlert:
 
+.. code-block:: bash
+
+  git clone --branch stable https://github.com/airbnb/streamalert.git
+
+2. Create and activate a virtual environment:
+
+.. code-block:: bash
+
+  cd streamalert
+  virtualenv -p python2.7 venv
+  source venv/bin/activate
+
+3. Install the StreamAlert requirements:
+
+.. code-block:: bash
+
+  pip install -r requirements.txt
+
+4. Run unit tests to make sure everything is installed correctly:
+
+.. code-block:: bash
+
+  tests/scripts/unit_tests.sh
+
+Set AWS Credentials
+-------------------
+
+1. Create an AWS account and an IAM user with permissions for at least the following services:
```

Comment:
can we link to AWS docs on this? ie: https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_create.html